docs/js_run_devserver.md

```diff
@@ -90,28 +88,16 @@ Supports running with [ibazel](https://github.com/bazelbuild/bazel-watcher).
 Only `data` files that change on incremental builds are synchronized when running with ibazel.
 
 
-**ATTRIBUTES**
```

Comment:
Were most of these params incorrect?

Reply:
They were all inherited from js_binary... now most are not re-documented and instead `**kwargs` just links to the js_binary docs:

```
All other args from <code>js_binary</code> except for <code>entry_point</code> which is set implicitly.<br><br><code>entry_point</code> is set implicitly by <code>js_run_devserver</code> and cannot be overridden.<br><br>See https://docs.aspect.build/rules/aspect_rules_js/docs/js_binary
```

We do the same thing in the `jest_test` macro.